Views/CreateSeedView.swift

```diff
@@ -136,7 +136,7 @@ class CreateSeedView: UIView, UITextFieldDelegate {
   @objc func closeView(sender: UIButton) {
     varietyTextField.resignFirstResponder()
     if sender == cancelButton {
-      specieButton.setTitle("Abricotier", for: .normal)
+      specieButton.setTitle("PRUNUS_ARMENIACA".localized, for: .normal)
```

Comment:
Why specifically "PRUNUS_ARMENIACA"? Is it just because it's the first of the list?